neqo-http3/src/connection_client.rs

```diff
@@ -5930,4 +5945,27 @@ mod tests {
         };
         assert!(server.conn.events().any(stop_sending_event));
     }
+
+    fn do_test_client_critical_stream_create_failed(max_stream: u64) {
+        let mut client = default_http3_client();
+        let mut server = TestServer::new();
+        server.set_max_uni_stream(max_stream);
+        let out = do_handshake_only(&mut client, &mut server);
+        client.process(out.dgram(), now());
+        assert_closed(&client, &Error::StreamLimitError);
+    }
+
+    #[test]
+    fn test_client_control_stream_create_failed() {
+        // Client fails to create a control stream, since server
```

Comment:
Rather than use `//` comments inside the function, it is probably better to use `///` comments above.